v20.2/orchestrate-a-local-cluster-with-kubernetes-insecure.md

```diff
@@ -17,8 +17,6 @@ This page walks you through a simple demonstration, using the open-source [Kuber
 To orchestrate a physically distributed cluster in production, see [Orchestrated Deployments](orchestration.html). To deploy a 30-day free CockroachCloud cluster instead of running CockroachDB yourself, see the [Quickstart](../cockroachcloud/quickstart.html).
 {{site.data.alerts.end}}
 
-{% include {{ page.version.version }}/prod-deployment/insecure-flag.md %}
```

Comment:
I think it's good to include this warning on the `orchestrate` pages.

Reply:
I left it on the prod orchestration pages. Just removed from the local version of the tutorial, the one that uses minikube. That ok?

Reply:
Oh, I missed that this is local. Yeah, it's fine to leave it out of this one.